src/queryParameters/extractSearchParameters.ts

```diff
@@ -0,0 +1,12 @@
+import apiSpecification from '../openapi.json';
+import type {
+  Request,
+} from 'express';
+
+export const extractSearchParameters = (
+  { query }: Pick<Request, 'query'>,
```

Comment:
Why `Pick<Request...>` and not just `Request`?

Reply:
Ah yes, at this point Request is fine given that there is no validation occurring -- in a world with validation this would have signaled that the request is expecting the request to have a `query` value, I think was the logic!